src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyle = {};
+
+  if (sourceString.trim() === '') {
+    return {};
+  }
+
+  const arrFromString = sourceString.trim().split(';');
+
+  const getKeyandProp = arrFromString.map((item) => {
+    const [key, prop] = item.split(':').map((part) => part.trim());
+
+    return { key, prop };
```

Comment:
You can directly assign the properties to 'objectOfStyle' within the 'map' callback function, eliminating the need for the 'forEach' loop on lines 23-27. This would make the code more concise and efficient.